tests/console/lvm.pm

```diff
@@ -38,11 +38,19 @@ sub run {
     assert_script_run 'echo -e "g\nn\n\n\n+1G\nt\n8e\nn\n\n\n+1G\nt\n2\n8e\nn\n\n\n\nt\n\n8e\np\nw" | fdisk ' . $disk;
     assert_script_run 'lsblk';
 
+    my $dev_name;
+    if (script_run("lsblk | grep vda1 | grep 1G") == 0 && script_run("lsblk | grep vda2 | grep 1G") == 0) {
```

Comment:
I wonder if we should check `$disk` value. Imagine `PLAYGROUNDDISK` is set to `sda` and we will replace it with `vdb`. To me it looks like that we should do something similar to what we have in `lib/btrfs_test.pm`.

For me it also looks like it won't fix the issue for arm, as we format `vda` above and only after that switch to appropriate disk.

Reply:
:thinking: what do mean by "something similar to `btrfs_test`" ? Because I see that there we are checking if the `PLAYGROUNDDISK` env var is set and, if not, we are doing some parsing.
But I'm getting the  `PLAYGROUNDDISK` env var using the `get_required_var` function, so the test would fail if the variable didn't exist.

Reply:
That's why I've said something similar, as you need to introduce more changes. But in general setting  PLAYGROUNDDISK in the test code would be fine. Better than setting it in the test suite and then overriding it in the test. 
But still, main issue with your code is that you use  `PLAYGROUNDDISK` value first and then change it to the other disk. As I've explained above, we format one disk and then you set it to other value. We should use same disk on both steps. Therefore, I've proposed to use same approach as in `btrfs_test` to detect the disk, as is should be unpartitioned in the original state.